### ISSUE TITLE
Resuming.

### ISSUE THREAD

**Issue by DarkAlchy** [TheLastBen/fast-stable-diffusion#323]
I trained with 3k steps and 15% (450 steps) of that for textual encoding.  I shut it down to come back and resume.  It shows I resumed but I really didn't.

![image](https://user-images.githubusercontent.com/10561690/199228447-12c52dfa-df74-4b40-a2f4-864dac1f886f.png)

It started over from scratch.

**Comment by TheLastBen** [comment]
It doesn't show the previous steps count, but it is resuming the training of the already trained model.

(not my business, but try dark mode in tools->settings, better for the eyes)

**Comment by DarkAlchy** [comment]
No, it really didn't resume because I am getting the same rather torn images at 500 and 1k as it started them completely over.  Even see where it redid the textual encoding again (freezing it).  I an say from all of what is shown, and the results gathered this did not really resume as it does if I stay connected and just go up manually.  If I disconnect and reconnect it blows up, even going so far as to have a completely different checksum than my previous every 500 steps for 3k steps have.
